src/udev/udev-rules.c

```diff
@@ -474,6 +474,13 @@ static int add_token(struct udev_rules *rules, struct token *token) {
         return 0;
 }
 
+static void log_unknown_owner(int error, const char *entity, const char *owner) {
+        if (IN_SET(error, -ENOENT, -ESRCH))
```

Comment:
I'd probably enclose the `error` in `abs()` in this case, just to make it idempotent regarding negative/non-negative error numbers. i.e. the way we do it for log_error_errno()... but i figure this doesn't matter too much